eng/Microsoft.Extensions.targets

```diff
@@ -1,6 +1,6 @@
 <Project>
   <PropertyGroup Condition="'$(TargetFramework.Contains(net6.0))' == 'true'">
-    <_MicrosoftHostingVersion>6.0.0-preview.3.21174.7</_MicrosoftHostingVersion>
+    <_MicrosoftHostingVersion>6.0.0-preview.4.21222.10</_MicrosoftHostingVersion>
```

Comment:
This version was chosen because it matches the chain of .NET 6 dependencies from the new version entry I added below. If these are mis-aligned, you'll get package downgrade errors.

Reply:
To put it another way, there may very well be newer versions (of preview4) in the feeds, but this is the version that matches the newest version of the ASP.NET Core / Blazor packages from preview4.